live-examples/css-examples/backgrounds-and-borders/border-radius.html

```diff
@@ -0,0 +1,72 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-radius">
+
+	<div class="example-choice" initial-choice="true">
+		<pre><code id="example_one" class="language-css">border-radius: 10px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+			<pre><code id="example_two" class="language-css">border-radius: 10px 5%;</code></pre>
+			<button type="button" class="copy hidden" aria-hidden="true">
+					<span class="visually-hidden">Copy to Clipboard</span>
+			</button>
+	</div>
+
+	<div class="example-choice">
+			<pre><code id="example_three" class="language-css">border-radius: 2px 4px 2px;</code></pre>
+			<button type="button" class="copy hidden" aria-hidden="true">
+					<span class="visually-hidden">Copy to Clipboard</span>
+			</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_four" class="language-css">border-radius: 1px 0 3px 4px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_five" class="language-css">border-radius: 10px 5% / 20px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_six" class="language-css">border-radius: 10px 5% / 20px 30px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_seven" class="language-css">border-radius: 10px 5px 2em / 20px 25px 30%;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_eight" class="language-css">border-radius: 10px 5% / 20px 25em 30px 35em;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_nine" class="language-css">border-radius: 100px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+</section>
+
+<div id="output" class="output large hidden">
+	<section id="default-example" class="default-example">
+		<div id="example-element" class="transition-all">Test Border Radius!</div>
```

Comment:
To be consistent with the other border-* properties (e.g. [`border-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-color)), let's have the text:

> "This is a box with rounded corners."